src/styles/components/modal/full-screen/styles.less

```diff
@@ -40,6 +42,11 @@
     .modal-body-offset {
       margin: -@modal-full-screen-body-padding-vertical -@modal-full-screen-body-padding-horizontal;
     }
+
+    .gm-scrollbar-container {
```

Comment:
@mperrotti removed this in https://github.com/dcos/dcos-ui/pull/4010 , but without it, our modal has 0 height on integration tests.